src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -5520,11 +5520,12 @@ void makeAndAdjustReceiverSlot(BoundExpression receiver)
             {
                 if (!wasTargetTyped)
                 {
-                    // This can happen when we're inferring the return type of a lambda. For this case, we don't need to do any work,
+                    // This can happen when we're inferring the return type of a lambda or visiting a suppressed node like
```

Comment:
>a suppressed node [](http://example.com/codeflow?start=100&length=17)

Not sure what this mean. Could you please elaborate?

Reply:
It means node visited without diagnostics, for example:

https://github.com/dotnet/roslyn/blob/2d503182bc7760b3aa5c7a8d5a8ee662dcc38852/src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs#L7588-L7590

```suggestion
                    // This can happen when we're inferring the return type of a lambda or visiting a node without diagnostics like
```